src/http/headers.cr

```diff
@@ -53,15 +54,18 @@ struct HTTP::Headers
     # the most common case is a single value and so we avoid allocating
     # memory for arrays.
     @hash = Hash(Key, String | Array(String)).new
+    @serialized = false
   end
 
   def []=(key, value : String)
+    check_already_serialized()
```

Comment:
```suggestion
    check_already_serialized
```

Reply:
I changed it. Why is it not changed by `crystal tool format`?

Reply:
This could potentially be a semantic change, so the formatter doesn't do it. Parenthesis disambiguates between accessign a local variable and calling a method if the identifier could be both.